Content.Server/Chemistry/EntitySystems/ChemistrySystem.Injector.cs

```diff
@@ -15,13 +15,22 @@
 using Robust.Shared.GameStates;
 using Robust.Shared.Player;
 using System.Threading;
+using Content.Shared.Verbs;
+using Robust.Server.GameObjects;
+using Content.Shared.Popups;
 
 namespace Content.Server.Chemistry.EntitySystems;
 
 public sealed partial class ChemistrySystem
 {
+
+    /// <summary>
+    ///     Default transfer amounts for the set-transfer verb.
+    /// </summary>
+    public static readonly List<int> DefaultTransferAmounts = new() { 5, 10, 15};
     private void InitializeInjector()
```

Comment:
```suggestion
{
    /// <summary>
    ///     Default transfer amounts for the set-transfer verb.
    /// </summary>
    public static readonly List<int> DefaultTransferAmounts = new() { 5, 10, 15};
    
    private void InitializeInjector()
```
Also maybe just call this `TransferAmounts`

Reply:
I don't see any difference in the suggested change diff. Updated the name as requested.

Reply:
Just whitespace. There should be a blank line before the first method.